TACC/Core-Styles
# fix: wp-63 must support two brands on login

## Overview

Support two logos on login form.

## Related

- [WP-63](https://jira.tacc.utexas.edu/browse/WP-63)
- required by https://github.com/tapis-project/authenticator/pull/36

## Changes

- add login form template option for two images
- add styles to layout two login form images
- add second login form example with two images

## Testing

1. Open these pattern demos:
	- http://localhost:3002/components/detail/c-form--login
    - http://localhost:3002/components/detail/s-form--login
    - http://localhost:3002/components/detail/s-form-page--login
2. Verify single-logo and double-logo forms otherwise look the same.
3. Test double-logo wraps on narrow viewport.

## UI

| pattern | one image | two images |
| - | - | - |
| `c-form` | <img width="992" alt="c-form one image" src="https://github.com/TACC/Core-Styles/assets/62723358/a019592a-5e06-4b1f-8e46-476b0b0e4ef1"> | <img width="992" alt="c-form two image" src="https://github.com/TACC/Core-Styles/assets/62723358/aeafca5a-0920-459e-b0ab-5340e15a405e">
| `s-form` | <img width="992" alt="s-form one image" src="https://github.com/TACC/Core-Styles/assets/62723358/f2a34721-6404-4eee-924a-2728da77c0e9"> | <img width="992" alt="s-form two image" src="https://github.com/TACC/Core-Styles/assets/62723358/e4720082-f354-4e73-bb27-5888c05873c5">
| `s-form-page` | <img width="992" alt="s-form-page one image" src="https://github.com/TACC/Core-Styles/assets/62723358/27d4761f-d077-480b-8cf6-c9c0b8fac3a9"> | <img width="992" alt="s-form-page two image" src="https://github.com/TACC/Core-Styles/assets/62723358/eeadf302-b968-45bd-a5f2-a12d3d72c1c0">

| logo side-by-side | logo wraps |
| - | - |
| <img width="992" alt="s-form-page two image" src="https://github.com/TACC/Core-Styles/assets/62723358/eeadf302-b968-45bd-a5f2-a12d3d72c1c0"> | <img width="992" alt="logo wrap at narrow nview" src="https://github.com/TACC/Core-Styles/assets/62723358/3f1828ce-e168-410b-9bdb-3332efc935da">
